### DISCUSSION TITLE
TeslaMate V1.26.2 coming soon?

### DISCUSSION THREAD

**Discussion by ktsenvolt** [teslamate-org/teslamate#2688] [General]
When is V1.26.2 coming? Current version 1.26.1 was released on Jan 28, 2022. Thank you.

**Comment by cwanja** [comment]
The software development answer is - when it is released. 

The human answer is - no one really knows beyond Adrian and anyone closely developing with him. You can infer that he is still developing, from his activity on GitHub in this repo and various other ones. So it seems, he has not abandon the software. But merely something else has taken a higher priority - personally or professionally. 

All that said, might I remind people to consider donating to Adrian. This is not his business and donations help. I treat this more like subscription software and less like freeware.

**Reply by ktsenvolt** [reply]
Thank you.

**Comment by tmushy** [comment]
Dont know if im reading things right but the packages.json file shows a 2.0 beta so maybe hes working on 2.0?
